### ISSUE TITLE
Ошибка при отображении блюда в dev af7a6278b4fff2bdff39353dfc363581c24df6a9

### ISSUE THREAD

**Issue by Alexey314** [cofeGB/cofeGB#35]
# При переходе по ссылке http://localhost:8080/menu/desserts/14512 в консоли возникает ошибка:
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Error in render: "TypeError: Cannot read properties of undefined (reading 'composition')"
found in
---> <Dish> at src/components/dish.vue
```
Проблемы две: 
 - в компонент dish передается prop item с undefined, 
 - компонент dish не работает корректно с item==undefined (на время загрузки данных о блюде?)

**Comment by PankratovaE** [comment]
в компонент dish передается prop item с undefined - была ошибка в шаблоне. Я поправила, сейчас у меня sandwich и starters работают.
А в десертах у нас пока ничего нет.

**Comment by Alexey314** [comment]
У нас роут
```
  {
    path: '/menu/:category/:id',
    name: 'dish',
    component: dish,
  },
  ```
  может включить dish c любым id из адресной строки, в т.ч. неверным/устаревшим

**Comment by elena3569** [comment]
не поняла, что имеешь в виду: обрабатывать неверные id?

**Comment by PankratovaE** [comment]
Да, сейчас получается, что если данных нет (по какой-то причине) просто весь компонент не отрисовывается. Лена, я поправлю.

**Comment by Alexey314** [comment]
да, при неверном id блюда, полученном из роута, dish не должен бы падать
